MarsMission/Controllers/ViewController.swift

```diff
@@ -9,9 +9,11 @@ import UIKit
 
 class ViewController: UIViewController {
 
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        navigationItem.title = "Weather Forecast"
```

Comment:
Localise this string

Reply:
<img width="532" alt="Screenshot 2020-11-05 at 05 55 52" src="https://user-images.githubusercontent.com/55865505/98195982-b2473d00-1f2b-11eb-864f-746bdb6eac15.png">